about/index.html

```diff
@@ -1,102 +1,141 @@
+						Discord server in order to provide more equity for students, serving an audience of 400 students
+						the latest school and club news.
 
-As the only VP candidate with SAC experience running in-person events, I'm also more able to prepare events like the Semi-Formal dance, MACtoberfest, etc (only 3 candidates have 2 years, and the others are candidates for presidents).
+						As the only VP candidate with SAC experience running in-person events, I'm also more able to
+						prepare events like the Semi-Formal dance, MACtoberfest, etc (only 3 candidates have 2 years,
+						and the others are candidates for presidents).
 
-Executive Position: SAC (2y), MCPT (1y), Athletic Council (3y), The Flounder (2y), The Lyon (1y), Arts Council (1y), Tech Team (3y). Team Member: Yearbook (1y), Drama Production (1y), Badminton (1y), Concert Band (2y), DECA (2y). Member: MSC (most points of any grade :D).</p>
-					</div>
+						Executive Position: SAC (2y), MCPT (1y), Athletic Council (3y), The Flounder (2y), The Lyon
+						(1y), Arts Council (1y), Tech Team (3y). Team Member: Yearbook (1y), Drama Production (1y),
+						Badminton (1y), Concert Band (2y), DECA (2y). Member: MSC (most points of any grade :D).</p>
 				</div>
 			</div>
-			<br/>
-			<br/>
 		</div>
+		<br />
+		<br />
+	</div>
 
-		<div class="footer">
-			<div class="footer-title-par"><p class="footer-title">This website was made from scratch by <span style="font-weight: 700">Justin Lu, Nicole Cui, Eric Shim, and Diana Lin</span> as a mock for an improved student website :D</p></div>
-			<hr class="footer-bar">
-			<div class="foooter-icon-par">
-				<a class="footer-name" href="https://www.instagram.com/cuilu.vp/" target="_blank"><img class="footer-icon" src="../img/insta-icon.svg"></a>
-				<a class="footer-name" href="https://discord.gg/DT6vdmPJw4" target="_blank"><img class="footer-icon" src="../img/discord-icon.svg"></a>
-				<a class="footer-name" href="mailto:cuilu@justinlu.ca" target="_blank"><img class="footer-icon" src="../img/email-icon.svg"></a>
-			</div>
+	<div class="footer">
+		<div class="footer-title-par">
+			<p class="footer-title">This website was made from scratch by <span style="font-weight: 700">Justin Lu,
+					Nicole Cui, Eric Shim, and Diana Lin</span> as a mock for an improved student website :D</p>
+		</div>
+		<hr class="footer-bar">
+		<div class="foooter-icon-par">
+			<a class="footer-name" href="https://www.instagram.com/cuilu.vp/" target="_blank"
+				rel="noopener noreferrer"><img class="footer-icon" src="../img/insta-icon.svg"></a>
+			<a class="footer-name" href="https://discord.gg/DT6vdmPJw4" target="_blank" rel="noopener noreferrer"><img
+					class="footer-icon" src="../img/discord-icon.svg"></a>
+			<a class="footer-name" href="mailto:cuilu@justinlu.ca" target="_blank" rel="noopener noreferrer"><img
+					class="footer-icon" src="../img/email-icon.svg"></a>
 		</div>
-	</body>
-</html>
+	</div>
+</body>
+
+</html>
```

Comment:
As someone who has 0 stake in anything here, please keep the newlines here (and everywhere else). They're nice.